app/controllers/api/v1/emotions_controller.rb

```diff
@@ -73,4 +77,12 @@ def emotion_params
   def time_period
     @time_period ||= TimePeriod.find_or_create_time_period
   end
+
+  def list_shoutouts_to(user)
+    shoutouts_to_user = ShoutoutRecipient.where(user_id: user.id)
```

Comment:
you can use `current_user.shoutout_recipients` instead of `ShoutoutRecipient.where(user_id: user.id)`

Reply:
Thanks. It had relationship with previous comment which I changed so it is not actual now